package.json

```diff
@@ -7,19 +7,26 @@
     "npm": ">= 8.3.2"
   },
   "dependencies": {
+    "@ac32/spookyswap-sdk": "1.0.0",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.1.1",
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.5.0",
     "@types/node": "^16.11.33",
-    "@types/react": "^18.0.8",
-    "@types/react-dom": "^18.0.3",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0",
+    "@types/react": "^17.0.39",
+    "@types/react-dom": "^17.0.9",
+    "@reduxjs/toolkit": "^1.4.0",
+    "ethers": "^5.6.5",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-redux": "^7.2.1",
```

Comment:
Why not using react-redux@8.0.01?

From [react-redux changelog](https://github.com/reduxjs/react-redux/releases/tag/v8.0.0)
>  React-Redux v8 is compatible with all React versions that have hooks (16.8+, and React Native 0.59+)

Reply:
not needed for now